src/docker/dockerfileParser.ts

```diff
@@ -0,0 +1,73 @@
+import * as fs from 'fs';
+import { CommandEntry, parse as rawDockerfileParser} from 'docker-file-parser';

```

Comment:
`parse as rawDockerfileParser` sounds like you're aliasing a verb to a noun, which feel like it's going to be misleading.  Can we find a better alias?  Do we need to alias it at all?